WebView.js

```diff
@@ -1,6 +1,9 @@
-import EventEmitter from 'events';
-import React, { Component } from 'react';
-import { WebView as NativeWebView } from 'react-native';
+import EventEmitter from "events";
+import React, { Component } from "react";
+import { WebView as NativeWebView } from "react-native";
```

Comment:
I'd leave single quotes everywhere .This is common and well known part of javascript styleguide. Apart of that, every project has it's own styleguide as well all developers have there own coding style/preferences. If smth seems really weird to one, this should be raised as discussion point. Codestyle changes shouldn't be a part of feature pull request/bug fix